src/App.tsx

```diff
@@ -34,62 +37,99 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort((g1, g2) => g1.localeCompare(g2));
+  } else if (sortType === SortType.LENGTH) {
```

Comment:
when we use the `else if` statement implies to have an `else` statement on the end. Like:
```js
if (condition) {
  // doSomething()
} else if (otherCondition) {
  // doOtherThing()
} else { // if any condition matches
  // doThis()
}

Reply:
We have only 2 kinds of sort. What logic do You expect on `else`?